Tests/SwiftLintFrameworkTests/ConfigurationTests.swift

```diff
@@ -89,11 +89,11 @@ class ConfigurationTests: XCTestCase {
         XCTAssertEqual(disabledConfig.disabledRules,
                        ["nesting", "todo"],
                        "initializing Configuration with valid rules in Dictionary should succeed")
-        let expectedIdentifiers = Array(masterRuleList.list.keys)
-            .filter({ !(["nesting", "todo"] + optInRules).contains($0) })
-        let configuredIdentifiers = disabledConfig.rules.map {
+        let expectedIdentifiers = Set(masterRuleList.list.keys
```

Comment:
I remembered now why I've changed to `Set`: it was to make sure the order doesn't matter. As you can see, 04fc177 is failing on Linux because of it:

```
/SwiftLint/Tests/SwiftLintFrameworkTests/ConfigurationTests.swift:118: error: ConfigurationTests.testDisabledRulesWithUnknownRule : XCTAssertEqual failed: ("["syntactic_sugar", "function_parameter_count", "statement_position", "unused_closure_parameter", "variable_name", "trailing_comma", "closing_brace", "trailing_whitespace", "trailing_newline", "custom_rules", "legacy_constant", "redundant_string_enum_value", "file_length", "class_delegate_protocol", "private_unit_test", "valid_docs", "vertical_parameter_alignment", "valid_ibinspectable", "comma", "empty_parameters", "unused_enumerated", "redundant_optional_initialization", "redundant_void_return", "line_length", "function_body_length", "implicit_getter", "mark", "operator_whitespace", "weak_delegate", "legacy_nsgeometry_functions", "todo", "type_body_length", "empty_parentheses_with_trailing_closure", "dynamic_inline", "force_try", "trailing_semicolon", "type_name", "vertical_whitespace", "leading_whitespace", "control_statement", "opening_brace", "colon", "force_cast", "legacy_constructor", "return_arrow_whitespace", "void_return", "legacy_cggeometry_functions", "cyclomatic_complexity", "closure_parameter_position"]") is not equal to ("["unused_closure_parameter", "function_parameter_count", "statement_position", "closure_parameter_position", "trailing_comma", "closing_brace", "trailing_whitespace", "trailing_newline", "custom_rules", "legacy_constant", "redundant_string_enum_value", "file_length", "class_delegate_protocol", "private_unit_test", "valid_docs", "vertical_parameter_alignment", "valid_ibinspectable", "comma", "empty_parameters", "unused_enumerated", "redundant_optional_initialization", "redundant_void_return", "function_body_length", "implicit_getter", "line_length", "mark", "operator_whitespace", "weak_delegate", "legacy_nsgeometry_functions", "todo", "type_body_length", "empty_parentheses_with_trailing_closure", "dynamic_inline", "force_try", "trailing_semicolon", "type_name", "vertical_whitespace", "leading_whitespace", "control_statement", "opening_brace", "colon", "force_cast", "legacy_constructor", "return_arrow_whitespace", "void_return", "syntactic_sugar", "variable_name", "legacy_cggeometry_functions", "cyclomatic_complexity"]") - 
```